### ISSUE TITLE
Translate schema comments ($comment) to C# documentation comments

### ISSUE THREAD

**Issue by Trolldemorted** [RicoSuter/NJsonSchema#1428]
Could you take the content of the $comment field and translate it into a C# doc comment?

**Comment by Purgator** [comment]
I strongly agree with this, this is how I expect it to work. I want to do the operation on the other side.
I want to create a json definition from a c# class.
I have just tried out the package and is amazing but, I wanted to have my xml comment translated out into a json description. This is the natural way to do.

**Comment by Purgator** [comment]
My bad, the way I want it works already. It is not based on reflexion but on the generated xml file.
Add         `<GenerateDocumentationFile>true</GenerateDocumentationFile>` to the csproj to generate the doc.
